ffmeta/templates/web/variable.html

```diff
@@ -134,13 +134,17 @@ <h4 class="panel-title">
                             <tr>
                                 <th>Value</th>
                                 <th>Label</th>
+                                <th align="right">Frequency</th>
+                                <th align="right">Percent</th>
                             </tr>
                         </thead>
                         <tbody>
                         {% for r in responses %}
                             <tr>
                                 <td>{{r.value}}</td>
                                 <td>{{r.label}}</td>
+                                <td align="center">{{r.freq}}</td>
```

Comment:
Your labels are right-aligned, but values are center-aligned? Probably best to not have explicit alignment at all (only useful if you have several rows of numeric values).

Reply:
I agree with you in theory, but when I tested this out the values did not look aligned.
<img width="315" alt="Screen Shot 2020-05-06 at 9 55 54 AM" src="https://user-images.githubusercontent.com/10812643/81185687-e4c5ac80-8f7f-11ea-8d1a-241130b3943c.png">